pandas/core/window/ewm.py

```diff
@@ -71,30 +72,48 @@ class EWM(_Rolling):
-    weights (1-alpha)**(n-1), (1-alpha)**(n-2), ..., 1-alpha, 1.
-
-    When adjust is False, weighted averages are calculated recursively as:
-       weighted_average[0] = arg[0];
-       weighted_average[i] = (1-alpha)*weighted_average[i-1] + alpha*arg[i].
-
-    When ignore_na is False (default), weights are based on absolute positions.
-    For example, the weights of x and y used in calculating the final weighted
-    average of [x, None, y] are (1-alpha)**2 and 1 (if adjust is True), and
-    (1-alpha)**2 and alpha (if adjust is False).
-
-    When ignore_na is True (reproducing pre-0.15.0 behavior), weights are based
-    on relative positions. For example, the weights of x and y used in
-    calculating the final weighted average of [x, None, y] are 1-alpha and 1
-    (if adjust is True), and 1-alpha and alpha (if adjust is False).
-
-    More details can be found at
-    https://pandas.pydata.org/pandas-docs/stable/user_guide/computation.html#exponentially-weighted-windows
+    Exactly one paramter: ``com``, ``span``, ``halflife``, or ``alpha``
+    must be provided.
+    Allowed values and the relation between the parameters are specified in the
+    parameter descriptions above (see the link at the end of this section for
+    a detailed explanation).
+
+    Available EW (exponentially weighted) methods: ``mean()``, ``var()``, ``std()``,
+    ``corr()``, ``cov()``.
+
+    When ``adjust=True`` (default), the EW function is calculated using
+    weights :math:`w_i = (1 - \alpha)^i`.
+    For example, the EW moving average of the series [:math:`x_0, x_1, ..., x_t`] would
+    be:
+
+    .. math::
+        y_t = \frac{x_t + (1 - \alpha)x_{t-1} + (1 - \alpha)^2 x_{t-2} + ...
+        + (1 - \alpha)^t x_0}{1 + (1 - \alpha) + (1 - \alpha)^2 + ... + (1 - \alpha)^t}
+
+    When ``adjust=False``, the exponentially weighted function is calculated
+    recursively:
+
+    .. math::
+        \begin{split}
+            y_0 &= x_0\\
+            y_t &= (1 - \alpha) y_{t-1} + \alpha x_t,
+        \end{split}
+
+    When ``ignore_na=False`` (default), weights are based on absolute positions.
+    For example, the weights of :math:`x_0` and :math:`x_2` used in calculating the
+    final weighted average of [:math:`x_0`, None, :math:`x_2`] are :math:`(1-\alpha)^2`
+    and :math:`1` if ``adjust=True``, and :math:`(1-\alpha)^2` and
+    :math:`\alpha` if ``adjust=False``.
+
+    When ``ignore_na=True`` (reproducing pre-0.15.0 behavior), weights are based on
+    relative positions. For example, the weights of :math:`x_0` and :math:`x_2` used in
+    calculating the final weighted average of [:math:`x_0`, None, :math:`x_2`] are
+    :math:`1-\alpha` and :math:`1` if ``adjust=True``, and :math:`1-\alpha` and
+    :math:`\alpha` if ``adjust=False``.
+
+    More details can be found at:
+    `Exponentially weighted windows
+    <https://pandas.pydata.org/pandas-docs/stable/user_guide/computation.html#exponentially-weighted-windows>`_.
```

Comment:
Sphinx allow you to create links automatically with `:ref:`. We should use this, since we may be changing the path of the docs (may be using the domain pandas.io, or using pandas.pydata.org/docs/ instead of pandas-docs/stable...).

Reply:
Tried that, didn't succeed. If you can instruct me here, I'd be thankful.

Reply:
If you grep for `:ref:` you can find examples, like:
```
:ref:`API Changes <whatsnew_0141.api>`
```
What's inside `<>` is a label somewhere in the code, defined as `.. _whatsnew_0141.api:`. If you don't have a label for the section you're linking here (it probably exists), you can just create it following the convention of the other labels in that page.

Reply:
I'm not able to make `:ref:` work. 
In the same file there is not label defined.
This is my first time working with Sphinx, that's why I'm asking for help.

Reply:
I don't think I can provide more information that what I said above. With that, and checking other examples, should be quite straight forward to get this working.

Reply:
Not sure what you tried, but this should be simply:
```
:ref:`Exponentially weighted windows <stats.moments.exponentially_weighted>`
```

Reply:
``` :ref:`Exponentially weighted windows <stats.moments.exponentially_weighted>` ```
Does not produce a link. Does not give me an error either. 
Could it be that I don't see it as a link because I build only the `pandas.DataFrame.ewm` page?